disperser/batcher/txn_manager.go

```diff
@@ -37,6 +37,10 @@ type TxnRequest struct {
 	Metadata interface{}
 
 	requestedAt time.Time
+	// txAttempts are the transactions that have been attempted to be mined for this request.
+	// If a transaction hasn't been confirmed within the timeout and a replacement transaction is sent,
+	// the original transaction hash will be kept in this slice
+	txAttempts []*types.Transaction
```

Comment:
If the batcher restarts, the blobs will be retried. Even if we persisted this, it would be difficult to recover the blobs given how we keep most of the state in memory. 
We can persist encoded blob store + transactions for optimization in the future.